src/data/effects.py

```
@@ -0,0 +1,315 @@
+    BUFF_COLD_UP = "BUFF_COLD_UP"
+    BUFF_BURN_UP = "BUFF_BURN_UP"
+    BUFF_ELECTROMAGNETIC_UP = "BUFF_ELECTROMAGNETIC_UP"
+    BUFF_NATURAL_UP = "BUFF_NATURAL_UP"
+
+    # 减益效果
+    DEBUFF_DEF_DOWN = "DEBUFF_DEF_DOWN"
+    DEBUFF_SPEED_DOWN = "DEBUFF_SPEED_DOWN"
+    DEBUFF_HEAL_DOWN = "DEBUFF_HEAL_DOWN"
+
+    # 放置物
+    PLACE_THUNDER_SPEAR = "PLACE_THUNDER_SPEAR"  # 雷枪（艾维文娜战技召回时造成伤害）
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)